source/docs/articles/sites/domains.md

```diff
@@ -33,15 +33,30 @@ You can simultaneously add both the bare domain name and the www subdomain. This
 ### Develop Using a Domain Without Changing DNS
 Sometimes it's useful to develop on a site using a specific domain, but the overhead of temporarily changing DNS is too much. Use the following workaround to allow your local workstation to access your Pantheon site by the desired domain without changing DNS. This requires a paid plan.
 
-1. From the Pantheon Dashboard, add the domain to the target site environment.
-2. Add a line to your local <a href="https://en.wikipedia.org/wiki/Hosts_(file)">Hosts file</a> with the provided IP address, default value for the Live environment, and the bare domain.
+<div class="alert alert-danger" role="alert">
+<h4>Important</h4>This process is for short-term DNS testing. Endpoint IPs will change over time. Be sure to use the Dashboard's recommendations when going Live.</div>
```

Comment:
@aeligature @nataliejeremy  Looking at this again...how about a stronger warning that mentions downtime? Maybe:

This process is for short-term testing only. Be sure to use the Dashboard's recommendations when going live, or you will experience downtime when IP addresses change.